src/library/commissioner_safe.cpp

```diff
@@ -91,20 +94,22 @@ const Config &CommissionerSafe::GetConfig() const
     return mImpl.GetConfig();
 }
 
-Error CommissionerSafe::Start()
+Error CommissionerSafe::StartEventLoopThread()
 {
     Error error = Error::kNone;
 
     VerifyOrExit(!mEventThread.joinable(), error = Error::kAlready);
```

Comment:
```suggestion
    ASSERT(!mEventThread.joinable());
```
Can we use ASSERT now since it can only be called once ?

Reply:
I have the same idea in previous comment. Will do it.

Reply:
@simonlingoogle 
changed `StartEventLoopThread` to return `void`, please review.